packages/core-data/src/entities.ts

```diff
@@ -310,6 +310,25 @@ const pluginConfig: PluginEntity[ 'config' ] = {
 	key: 'plugin',
 };
 
+type NavigationEntity< C extends Context = Context > = EntityType<
+	{
+		name: 'navigation';
+		kind: 'root';
+		baseURLParams: { context: 'edit' };
+		key: 'slug';
+	},
+	Records.Plugin< C >,
```

Comment:
I'll update this _if_ we go ahead with this route.

Reply:
That’s gone, just rebase this pr

Reply:
Oh you mean literally the whole file has been removed 😆